SpaceDash/Controller/ViewControllers/HomeViewController.swift

```diff
@@ -78,12 +84,31 @@ class HomeViewController: UIViewController,NetworkManagerDelegate {
         launchDate.text =  upcomingLaunch?.getDate()
         
         self.isTentative.isHidden = !(self.upcomingLaunch?.decodedData!.is_tentative)!
-        
+        self.isTentative.isHidden = false
```

Comment:
what's this for?

Reply:
for debugging i added that. will remove it.